raw-data/EA/README.md

```diff
@@ -0,0 +1,18 @@
+# EA
+
+The `EA` source data comes from The database of Eurasian phonological inventories (beta version), online: [http://eurasianphonology.info/](http://eurasianphonology.info/).
+
+This source is cited as:
+
+> Nikolaev, Dmitry; Andrey Nikulin; and Anton Kukhto. 2015. The database of Eurasian phonological inventories. Available online at [http://eurasianphonology.info/](http://eurasianphonology.info/).
+
+The raw data and source code are available at: [https://github.com/macleginn/eurasian-phonologies](https://github.com/macleginn/eurasian-phonologies).
+
+`EA` contains a fresh take on phonological inventories from Eurasia. The data were extracted from grammatical descriptions of individual language varieties. It contains phonemes and information about tone.
```

Comment:
```suggestion
`EA` contains a fresh take on phonological inventories from Eurasia. The data were extracted from grammatical descriptions of individual language varieties. It contains only phonemes, with no information on allophones or linguistic tone.
```

Reply:
I don't see the info about tone in the raw data file... am I missing something?  Also when looking for the tone info, I noticed some issues with the raw data (mixing vowels and glides, like `aw` instead of `au̯`, some TODOs in the correspondences file)... should probably open a new issue to deal with those.